voxel51/cli/cli.py

```diff
@@ -1303,12 +1326,15 @@ def setup(parser):
         parser.add_argument(
             "--sort-by", metavar="FIELD",
             help="field to sort by when listing analytics")
-        parser.add_argument(
-            "-a", "--all-versions", action="store_true",
-            help="whether to include all versions of analytics")
         parser.add_argument(
             "--ascending", action="store_true",
             help="whether to sort in ascending order")
+        parser.add_argument(
+            "--all-versions", action="store_true",
+            help="whether to include all versions of analytics")
+        parser.add_argument(
+            "-a", "--all-fields", action="store_true",
```

Comment:
Yeah I switched because I didn't want `-a` to mean `--all-fields` for data/jobs but `--all-versions` for analytics. And I didn't want to _not_ provide a shorthand for `--all-fields` for data/jobs. So, this is where I wound up